esphome/components/rf_bridge/rf_bridge.cpp

```diff
@@ -0,0 +1,100 @@
+#include "rf_bridge.h"
+#include "esphome/core/log.h"
+#include <cstring>
+
+namespace esphome {
+namespace rf_bridge {
+
+static const char* TAG = "rf_bridge";
```

Comment:
```suggestion
static const char *TAG = "rf_bridge";
```

Don't know why travis CI is acting up again, but this is likely required to make format checks pass.